src/components/logo.js

```diff
@@ -1,28 +1,77 @@
 import React from "react"
-import { StaticQuery, graphql } from "gatsby"
-import Img from "gatsby-image"
+import styled from "styled-components"
 
-const Logo = ({ className, children }) => (
-  <StaticQuery
-    query={graphql`
-      query {
-        placeholderImage: file(relativePath: { eq: "sandbox-banner.png" }) {
-          childImageSharp {
-            fluid(maxHeight: 40) {
-              ...GatsbyImageSharpFluid_tracedSVG
-            }
-          }
-        }
-      }
-    `}
-    render={data => (
-      <Img
-        className={className}
-        fluid={data.placeholderImage.childImageSharp.fluid}
+const StyledSVG = styled.svg`
+  #Path {
+    transition: fill 0.3s;
+    fill: ${props => props.fillColor};
+  }
+  #Shadow {
+    transition: fill 0.3s;
+    fill: ${props => props.shadowColor};
+  }
+`
+
+const Logo = ({ size = "20px", color = "white" }) => {
```

Comment:
React throws the following warning due to the Logo usage in the footer. 
```
It looks like you've wrapped styled() around your React component (Logo),
but the className prop is not being passed down to a child. 
No styles will be rendered unless className is composed within your React component.
```
We should probably make the change in the footer, but this will be a weird merge with the footer branch.